ci-operator/step-registry/ovn/conf/internal-masquerade-subnet/ovn-conf-internal-masquerade-subnet-commands.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+cat >> "${SHARED_DIR}/manifest_cluster-network-03-config.yml" << EOF
+apiVersion: operator.openshift.io/v1
+kind: Network
+metadata:
+  name: cluster
+spec:
+  defaultNetwork:
+    ovnKubernetesConfig:
+      gatewayConfig:
+        ipv4:
+          internalMasqueradeSubnet: 100.254.169.0/29
+EOF
+
+cat ${SHARED_DIR}/manifest_cluster-network-03-config.yml
```

Comment:
A lot of the files don't end with an empty new line, unsure if that's an issue

Reply:
no, I don't think it's an issue as the jobs work as expected. I've seen both with and without I think. not sure if we have a standard to follow on this.